src/test/java/uk/gov/dhsc/htbhf/steps/CommonSteps.java

```diff
@@ -97,18 +97,32 @@ protected void selectTextOnSendCode() {
         sendCodePage.clickContinue();
     }
 
+    protected void selectEmailOnSendCode() {
```

Comment:
Not worth updating this PR for on its own, but should this method be in ConfirmationCodeNavigationSteps?

Reply:
Maybe, but as you say, it's not worthwhile updating this PR for it.